src/compiler/crystal/semantic/flags.cr

```diff
@@ -36,9 +36,11 @@ class Crystal::Program
     set.add "x86_64" if set.any?(&.starts_with?("amd64"))
     set.add "i686" if set.any? { |flag| %w(i586 i486 i386).includes?(flag) }
 
+    set.add "musl" if set.includes?("musleabihf")
```

Comment:
you should probably add "gnu" if it includes gnueabihf, just to make this symmetrical. It's not currently used but it may be in the future.